Source/WTF/wtf/cf/URLCF.cpp

```diff
@@ -73,18 +73,16 @@ RetainPtr<CFURLRef> URL::createCFURL() const
     return result;
 }
 
+#if !PLATFORM(WIN)
 String URL::fileSystemPath() const
 {
     auto cfURL = createCFURL();
     if (!cfURL)
         return String();
 
-#if PLATFORM(WIN)
-    CFURLPathStyle pathStyle = kCFURLWindowsPathStyle;
-#else
     CFURLPathStyle pathStyle = kCFURLPOSIXPathStyle;
```

Comment:
Nit: I suppose this no longer needs to be a local variable.

Reply:
Fixed.